zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/VariablesQueryControllerTest.java

```diff
@@ -157,11 +157,7 @@ void shouldSearchVariableWithSorting() {
                 "sort": [
                     {
                         "field": "name",
-                        "order": "desc"
-                    },
-                    {
-                        "field": "value",
-                        "order": "asc"
+                        "order": "DESC"
```

Comment:
❓ Why are we removing this from this test case?

Reply:
I have re-added this test, but with `asc` lower case in order to test the proeprty 

`<useEnumCaseInsensitive>true</useEnumCaseInsensitive>` from OpenAPI generator , from [here](https://github.com/OpenAPITools/openapi-generator/pull/15273/files)

(Use `MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS` does not work for OpenApi Generator on Jackson Mapper Config)

Reply:
Works for me 👍 
Thanks for looking into this, this will make the API a lot nicer to use and less error-prone 🚀